### PR TITLE
'solution'

### PR DESCRIPTION
**Pull request by IvanDeiko**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      title: title,
+      priority: priority,
+      id: this.id++
+    }
+
+    this.listItems.push(newItem);
+  }
+
+  removeItem(id) {
+   let index = this.listItems.findIndex((elem) => {
```
Comment (gestap) [inline src/main.js]:
```suggestion
   const index = this.listItems.findIndex(elem => elem.id === id);
```

File: src/main.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+     return elem.id === id;
+   });
+   this.listItems.splice(index, 1);
+  }
+
+  getItem(id) {
+    return this.listItems.find((elem) => {
```
Comment (gestap) [inline src/main.js]:
```suggestion
    return this.listItems.find(elem => elem.id === id);
```

File: src/main.js [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  next() {
+    const maxPriorityItem = Math.max(...this.listItems.map(todo => todo.priority));
+
+    if (!this.listItems.length || maxPriorityItem === NaN) {
+      throw new Error ('Empty array: there are no items in the list');
+    } 
+
+    return this.listItems.find(todo => todo.priority === maxPriorityItem);;
+  } 
+}
```
Comment (gestap) [inline src/main.js]: the document must end with one empty line

File: src/main.js [hunk 4]
```diff
@@ -0,0 +1,40 @@
+"use strict"
```
Comment (gestap) [inline src/main.js]:
```suggestion
"use strict"

```

**Review by ost-ptk — CHANGES_REQUESTED** [review]

**Review by ost-ptk — APPROVED** [review]

**Pull request closed without merging**